src/components/Organizations/Organizations.tsx

```diff
@@ -508,109 +507,123 @@ export function Organizations() {
-      <DialogOrgEdit isOpen={dialogOrgEditOpen} setOpen={setDialogOrgEditOpen} />
-      <DialogRemoveOrgs isOpen={dialogOrgRemoveOpen} setOpen={setDialogOrgRemoveOpen} />
-
-      <Box margin={'0px 20px 0px 20px'}>
-        {/* вызов панели "добавление организации" */}
-        <Box
-          sx={{
-            display: 'flex',
-            flexDirection: 'row',
-            alignItems: 'center',
-            gap: '18px',
-          }}>
-          <Fab
-            color="primary"
-            sx={{ margin: '20px 0px 20px 0px' }}
-            onClick={() => {
-              setDialogOrgAddOpen(true);
-            }}>
-            <AddIcon />
-          </Fab>
-          <Typography sx={{ fontSize: '20px' }}>Организация</Typography>
-        </Box>
-
-        <Paper sx={{ width: '100%' }}>
-          {/* панель инструментов */}
-          <Toolbar
-            sx={{
-              display: 'flex',
-              flexDirection: 'row',
-              justifyContent: 'space-between',
-              alignItems: 'flex-end',
-              '&.MuiToolbar-root': { padding: '5px' },
-            }}>
-            {/* панель поиска */}
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: 'row',
-                gap: '12px',
-                alignItems: 'flex-end',
-              }}>
-              <SearchIcon sx={{ ml: '5px' }} />
-              <TextField
-                variant="standard"
-                label="Поиск"
-                onChange={(event) => {
-                  handleSearchChange(event.target.value.trim());
-                }}
-              />
-              {findedCount >= 0 ? (
-                <Typography variant="body2" sx={{ mr: '20px' }}>
-                  Найдено {findedCount} записей
-                </Typography>
-              ) : null}
-            </Box>
+    <Observer>
+      {() => {
+        return (
+          <>
+            <DialogOrgAdd isOpen={dialogOrgAddOpen} setOpen={setDialogOrgAddOpen} />
+            <DialogOrgEdit isOpen={dialogOrgEditOpen} setOpen={setDialogOrgEditOpen} />
+            <DialogRemoveOrgs
+              isOpen={dialogOrgRemoveOpen}
+              setOpen={setDialogOrgRemoveOpen}
+            />
 
-            {/* панель редактирования */}
-            <Box sx={{ display: 'flex', flexDirection: 'row', alignItems: 'flex-end' }}>
-              {selectedCount ? (
-                <Typography variant="body2" sx={{ mr: '20px' }}>
-                  {' '}
-                  Выбрано {selectedCount} записей{' '}
-                </Typography>
-              ) : null}
-
-              <Tooltip title="Изменить выбранное" placement="top">
-                <span>
-                  <IconButton
-                    disabled={selectedCount !== 1}
-                    onClick={() => {
-                      setDialogOrgEditOpen(true);
+            <Box margin={'0px 20px 0px 20px'}>
```

Comment:
use `Stack` ([link](https://mui.com/material-ui/react-stack/)) instead of margins